demisto_sdk/commands/download/downloader.py

```diff
@@ -189,7 +189,7 @@ def handle_list_files_flag(self) -> bool:
         """
         if self.list_files:
             self.all_custom_content_objects = self.get_custom_content_objects()
-            list_files = [[cco['name'], cco['entity'][:-1]] for cco in self.all_custom_content_objects
+            list_files = [[cco['name'], cco['type']] for cco in self.all_custom_content_objects
```

Comment:
why did we change this? what is in `'type'`?

Reply:
what we get from `find_type` function. we changed it because its much more accurate to use the file type than the entity dir. for example: mapper & classifier are in the same entity dir (Classifiers) but have diff type